caluma/caluma_analytics/sql.py

```diff
@@ -85,7 +87,7 @@ def self_alias(self):
         this query.
         """
         if isinstance(self.from_, str):
-            return self.from_
+            return self.from_  # pragma: no cover
```

Comment:
Why isn't this covered anymore?

Reply:
This code path was used when referencing the DB table table directly, but now we're always going via the queryset from the visibilities, so `self.from_` is always a queryset/subquery. I'd like to keep it in for now, but might decide later to remove this completely.

Reply:
Add a comment for that

Reply:
Done